mojo-data/mojos/rulefit/cars/v1.00/rulefit_v1.00_cars_scores2.txt

```diff
@@ -0,0 +1,32 @@
+[23.822953025110316, 0.0]
+[-4.739043183167723, 0.0]
+[-7.7323700901091925, 0.0]
+[-1.7457162762262537, 0.0]
+[-1.7457162762262537, 0.0]
+[-1.215333685560985, 0.0]
+[-1.1497147244280064, 0.0]
+[-1.2009172357097668, 0.0]
+[-19.7277468728987, 0.0]
+[-19.849353804903537, 0.0]
+[22.271889091589216, 0.0]
+[3.577344068603199E10, 0.0]
+[3.577344069077104E20, 0.0]
+[3.5773440690771036E30, 0.0]
+[3.5773440690771037E40, 0.0]
+[3.5773440690771037E50, 0.0]
+[3.577344069077104E60, 0.0]
+[3.577344069077104E70, 0.0]
+[3.577344069077103E80, 0.0]
+[3.577344069077104E90, 0.0]
+[3.577344069077103E100, 0.0]
+[3.5773440690771036E110, 0.0]
+[3.577344069077104E120, 0.0]
+[3.577344069077104E130, 0.0]
+[3.5773440690771036E140, 0.0]
+[3.5773440690771034E150, 0.0]
+[3.5773440690771035E160, 0.0]
+[3.577344069077104E170, 0.0]
+[3.577344069077104E180, 0.0]
+[3.5773440690771035E190, 0.0]
+[3.577344069077104E200, 0.0]
+[-4.739043183167723]
```

Comment:
here is just one number because this is a test case when prediction vec is too short ([0] instead of [0,0]). this behaviour is consistent with other algos so this is ok.